sitemedia/js/components/books/BooksSearch.js

```diff
@@ -0,0 +1,166 @@
+import { mapGetters, mapActions, mapState, mapMutations } from 'vuex'
+import isEqual from 'lodash/isEmpty'
+
+import SearchFacet from '../SearchFacet'
+import SearchFilter from '../SearchFilter'
+import SearchSort from '../SearchSort'
+import { isEmpty } from '../../../../node_modules/rxjs/operators';
+
+export default Vue.component('BooksSearch', {
+    template: `
+    <div class="books-search">
+        <sui-container textAlign="center" text>
+            <h4 is="sui-header" class="results-count">
+            Displaying {{ totalResults }} {{ totalResults | pluralize('book') }}
+            </h4>
+        </sui-container>
+        <form class="search-form ui form">
+            <sui-menu tabular attached="top">
+                <div
+                    is="sui-menu-item"
+                    v-for="(tab, index) in tabs"
+                    :key="index"
+                    :active="activeTab === index"
+                    :content="tabLabel(tab)"
+                    @click="activeTab = index"
+                />
+            </sui-menu>
+            <sui-segment
+                v-for="(tab, index) in tabs"
+                :key="index"
+                :class="{ active: activeTab === index }"
+                class="bottom attached tab"
+            >
+                <div class="ui equal width stackable grid">
+                    <search-facet
+                        v-for="facetName in tab"
+                        v-bind="facetConfig.find(facet => facet.name === facetName)"
+                        :choices="facetChoices.filter(choice => choice.facet === facetName)"
+                        :key="facetName"
+                    >
+                    </search-facet>
+                </div>
+            </sui-segment>
+            <sui-segment inverted class="active-items">
```

Comment:
Should only display when there are active items, right?

Reply:
Do we have a design of the form with no options active? The spec for the Annotations search shows one without this section displayed...should it just go away completely when nothing is active?